projects/README.md

```diff
@@ -0,0 +1,79 @@
+# Projects
+
+## angular/angular-proto-firestore
+An template project that shows how to use Angular + Protocol Buffers.
+
+See [README](angular/angular-proto-firestore/README.md).
```

Comment:
At the moment the projects have no README. Link for future updates?

Reply:
Yup